Dockerfile

```diff
@@ -39,3 +39,6 @@ COPY . $DIR
 RUN cd bashbrew/go && gb build
 
 VOLUME $BASHBREW_CACHE
+
+RUN ln -s "$PWD/bashbrew/entrypoint.sh" /usr/local/bin/bashbrew-entrypoint.sh
+ENTRYPOINT ["docker-entrypoint.sh"]
```

Comment:
`s/docker-entrypoint/bashbrew-entrypoint/`

Also the new `docker-entrypoint.sh` needs to be moved and renamed so the `ln` works. :wink: